templates/base.html

```diff
@@ -73,6 +73,7 @@
         <meta name="msapplication-TileImage" content="{{ STATIC_URL }}mstile-144x144.png">
     </head>
     <body class="{% if profile %}{% if is_external_student %}external-student{% else %}internal-student{% endif %}{% endif %} lang-{{ request.LANGUAGE_CODE }}" data-view-tag="{% block view_tag %}{% endblock %}">
+        <a href="#content" class="skip-link">{% trans "Skip to content" %}</a>
```

Comment:
When clicking this link, it scrolls me down and moves me to the content (past the navigation up top) to this view:
![image](https://user-images.githubusercontent.com/50318434/84667494-1d678700-af2b-11ea-8cc9-d2373b0ec8f2.png)

However, when I press tab again, it scrolls me up a bit and displays the "Skip course menu" button. Can this unnecessary scrolling up and down be avoided?
![image](https://user-images.githubusercontent.com/50318434/84667998-b8f8f780-af2b-11ea-9a3d-e8c419c223ee.png)

Reply:
Hi @etanttila ,

I've tried to replicate this, and haven't been able to. It seems to work as expected (top bar is always visible, scrolling is to just the top of the content area) on Chrome, Firefox and Safari on macOS. I've had a bit of trouble getting a local version tested in Ubuntu so far.

Reply:
I think the code works correctly for me too. Yesterday, I though the first link was supposed to take you to the course-content, but I was mistaken.

Reply:
The problem doesn't exist for me on Chrome (clicking on the link on pages with a course menu scrolls to just below the blue top bar so the blue top bar gets hidden), but does happen every time on Firefox (version 77.0.1 on Ubuntu) that there is a course menu (so not on the front page nor login page, on those pages the blue top bar stays up top). Strange, I wonder what causes this.

Reply:
I wonder if one of the other summer interns could check if they have the same problem or it it's just me. For example @saarasat